### ISSUE TITLE
Add documentation for "ignore_major_updates" config

### ISSUE THREAD

**Issue by Turnerj** [plastyk/silverstripe-dashboard#115]
We don't actually have anything that explains how it works in the documentation that I could see. Worth expanding on how it works by setting it to `false` or putting a date value in.

**Comment by 3Dgoo** [comment]
Yeah, definitely. Can I leave this up to you or Nick to do? As I don't actually know how this works.

**Comment by Turnerj** [comment]
Yeah, sure thing. Just wanted to add it hear as I was talking to Ruth about this feature today and realised we haven't covered it in the documentation.
